openreview/tools.py

```diff
@@ -489,9 +490,16 @@ def replace_members_with_ids(client, group):
                 else:
                     raise e
         else:
-            profile = client.get_profile(member)
-            ids.append(profile.id)
+            try:
```

Comment:
there is a function in tools that does exactly the same. It returns `None` if profile can not be found.

Reply:
Agreed @melisabok , this change has been made.